api/src/main/java/com/coinbase/exchange/api/accounts/AccountHistory.java

```diff
@@ -3,12 +3,13 @@
 import com.coinbase.exchange.model.Detail;
 
 import java.math.BigDecimal;
+import java.math.BigInteger;
 
 /**
  * Created by irufus on 2/18/15.
  */
 public class AccountHistory {
-    private Integer id;
+    private BigInteger id;
```

Comment:
not sure if this is a good idea to use BigInteger, why not just use String and we can adopt any future changes from Coinbase.